repairnator/src/main/java/fr/inria/spirals/repairnator/serializer/json/JsonSerializer.java

```diff
@@ -310,6 +332,10 @@ public void serializeData(ProjectInspector inspector) {
             case PATCHED:
                 outputHasBeenPatchedInspector(inspector);
                 break;
+                
+            case FIXERBUILD:
+            	outputFixerBuildInspector(inspector);
```

Comment:
I changed my mind. Regarding the serializers, I made changes only in the CSV serializer for now by creating the `CSVSerializerUtils` class and then we have two CSVSerializer, one for Bears and other for RepairNator.